integration/bin/tachyon-yarn-submit.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Usage:
+#  tachyon-yarn-submit.sh <numWorkers> <pathHdfs>
+
+function printUsage {
+  echo "Usage: tachyon-yarn-submit.sh <numWorkers> <pathHdfs>"
+  echo -e "  numWorkers        \tNumber of Tachyon workers to launch"
+  echo -e "  pathHdfs          \tpath on HDFS to put tachyon jar and distribute it to YARN"
+  echo
+  echo "Example: ./tachyon-yarn-submit.sh 10 hdfs://localhost:9000/tmp/"
+}
+
+if [ "$#" != 2 ]; then
```

Comment:
put body of non-trivial script inside of a `main()` function (https://google.github.io/styleguide/shell.xml#main)